### PR TITLE
Updated File.Create

### PR DESCRIPTION
**Pull request by JeroenOortwijn**
## Summary

Added missing buffer size information and made the wording of all four summaries the same.
Also made the wording of all four path parameter descriptions the same.

### CODE REVIEW

File: xml/System.IO/File.xml [hunk 1]
```diff
@@ -1120,10 +1120,10 @@
...
       </Parameters>
       <Docs>
-        <param name="path">The name of the file.</param>
+        <param name="path">The path and name of the file to create.</param>
         <param name="bufferSize">The number of bytes buffered for reads and writes to the file.</param>
         <param name="options">One of the <see cref="T:System.IO.FileOptions" /> values that describes how to create or overwrite the file.</param>
-        <summary>Creates or overwrites the specified file, specifying a buffer size and a <see cref="T:System.IO.FileOptions" /> value that describes how to create or overwrite the file.</summary>
+        <summary>Creates or overwrites a file in the specified path, specifying a buffer size and a <see cref="T:System.IO.FileOptions" /> value that describes how to create or overwrite the file.</summary>
```
Comment (rpetrusha) [inline xml/System.IO/File.xml]:
```suggestion
        <summary>Creates or overwrites a file in the specified path, specifying a buffer size and options that describe how to create or overwrite the file.</summary>
```

File: xml/System.IO/File.xml [hunk 2]
```diff
@@ -1189,11 +1189,11 @@
...
       </Parameters>
       <Docs>
-        <param name="path">The name of the file.</param>
+        <param name="path">The path and name of the file to create.</param>
         <param name="bufferSize">The number of bytes buffered for reads and writes to the file.</param>
         <param name="options">One of the <see cref="T:System.IO.FileOptions" /> values that describes how to create or overwrite the file.</param>
         <param name="fileSecurity">One of the <see cref="T:System.Security.AccessControl.FileSecurity" /> values that determines the access control and audit security for the file.</param>
-        <summary>Creates or overwrites the specified file with the specified buffer size, file options, and file security.</summary>
+        <summary>Creates or overwrites a file in the specified path, specifying a buffer size, a <see cref="T:System.IO.FileOptions" /> value that describes how to create or overwrite the file and a <see cref="T:System.Security.AccessControl.FileSecurity" /> value that determines the access control and audit security for the file.</summary>
```
Comment (rpetrusha) [inline xml/System.IO/File.xml]:
```suggestion
        <summary>Creates or overwrites a file in the specified path, specifying a buffer size, options that describe how to create or overwrite the file, and a value that determines the access control and audit security for the file.</summary>
```

Comment (JeroenOortwijn) [reply xml/System.IO/File.xml]: Hello @rpetrusha, I agree with your suggestions, but I noticed a small discrepancy between them:
In this suggestion, you added a comma before "and", but in the other suggestion you didn't add a comma.
Should both have a comma before "and"?

Comment (rpetrusha) [reply xml/System.IO/File.xml]: No. Only the second should have a comma, @JeroenOortwijn.

Comment (JeroenOortwijn) [reply xml/System.IO/File.xml]: Ah, ok!

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @JeroenOortwijn. I'll merge your PR now. The changes should be live on docs.microsoft.com in the next day or two.

**Pull request merged**